notifique/src/main/java/com/nathanrassi/notifique/Notifique.java

```diff
@@ -15,14 +15,21 @@
   }
 
   @PrimaryKey(autoGenerate = true) final long id;
-  final String message;
+  private final String message;
+  private final String title;
+  private final String notifPackage;
+  private final Long timestamp;
```

Comment:
why private?
Room won't compile without being able to get these fields, and I think our app will want access to them as well (populating some UI with this data, probably).